components/suse-branding/src/login.scss

```diff
@@ -116,6 +141,79 @@ $login-label-color: #e0e0e0;
           }
         }
       }
+
+      .login-footer {
+        position: fixed;
+        bottom: 12px;
+        animation: fadein 2s ease;
+
+        @keyframes fadein {
+          0% { opacity: 0; }
+          50% { opacity: 0; }
+          100% { opacity: 1; }
+        }        
+
+        .copyright-and-links {
+          align-items: flex-end;
+
+          .copyright .product-name > span {
+            color: $suse-text-gray;
+            font-weight: $suse-login-medium-weight;
+          }
+        }
+        .company-logo {
+          width: 96px;
+        }
+      }
     }
   }
 }
+
+// We don't want form to scroll, so switch the flex so that the info panel gets less space
+@media screen and (max-width: 940px) {
+  .brand  .login-outer {
+    .login-page {
+      .login-background {
+        flex: 0.5;
+      }
+    }
+  }
+}
+
+// Even smaller - hide the info panel and move the produce name to the top of the page
+@media screen and (max-width: 720px) {
```

Comment:
This value seems a bit low, should it be 820 (customer centre is about 770 but they shrink their input fields)?